src/test/java/org/openmrs/contrib/qaframework/automation/AttachmentsSteps.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class AttachmentsSteps extends Steps {
+
+    @Given ("system launches attachments page")
+    public void lunchAttachementPage(){
```

Comment:
As we wait for the methods implementations on this pr,i wanted to point out a better ticket description ie RATEST-68:Write attachments steps than RATEST-68 Write attachments steps